openpathsampling/volume.py

```diff
@@ -419,7 +422,16 @@ def __init__(self, collectivevariable, lambda_min = 0.0, lambda_max = 1.0,
 
     def do_wrap(self, value):
         """Wraps `value` into the periodic domain."""
+
+        # this looks strange and mimics the modulo operation `%` while
+        # being folly compatible for simtk numbers and plain python as well
```

Comment:
"fully compatible" not "folly compatible" (you might say that simtk numbers not supporting % is folly, though!)